AircraftConfig/acconfig.nas

```diff
@@ -194,6 +195,11 @@ setlistener("/sim/signals/fdm-initialized", func {
 	{
 		save.restore(save.default, getprop("/sim/fg-home") ~ "/Export/" ~ getprop("/sim/aircraft") ~ "-save.xml");
 	}
+	
+	if (getprop("/options/system/fo-view") == 1) {
+		setprop("/sim/current-view/view-number", 8);
```

Comment:
Hard coding the view number isn't the best idea cause that might break it in future.

Reply:
Thanks for the answer

> Doesn't seem to work here.
> 
> I've ticked the checkbox and switched views forth and back (using the keys v and Ctrl+v) but I keep returning to the captains view.

Have I understood the feature request wrong? I just made that the copilot view is selected at the start. What did you mean instead, or doesn't that work for you?




> Hard coding the view number isn't the best idea cause that might break it in future.

I'll try to find a better way if that's ok.

Reply:
I think it works on initial start, but Ctrl-V afterwards, will return you to captain view, not first officer view.

It should be possible to use nasal to swap the y-axis position / name of the first officer - captain views? 🤔

Reply:
All right. Thanks for the hint. I'm working on it.

Reply:
As for avoiding hard coding the view number, take a look at the `view.indexof` function.

Reply:
And I'd like to propose changing the View names to `Pilot Flying` and `Pilot Monitoring`... if that works with your solution.